django/db/models/sql/where.py

```diff
@@ -76,6 +77,9 @@ def as_sql(self, compiler, connection):
         else:
             full_needed, empty_needed = 1, len(self.children)
 
+        if self.connector == XOR and not connection.features.supports_logical_xor:
+            raise TypeError('The XOR operator is not supported on this database backend')
```

Comment:
Please add a period:

```suggestion
            raise TypeError('The XOR operator is not supported on this database backend.')
```